pydocx/test/utils.py

```diff
@@ -282,8 +282,8 @@ def __init__(self, *args, **kwargs):
         super(XMLDocx2Html, self).__init__(path=None, *args, **kwargs)
 
     def _load(self):
-        self.document = WordprocessingDocument(path=None)
-        package = self.document.package
+        document = WordprocessingDocument(path=None)
```

Comment:
What I think I'd like to do is replace all of this logic with a WordprocessingDocumentFactory, but that would require updating a bunch of tests since the spec for that is somewhat different than what's being used here.